src/pybind/mgr/dashboard/tests/test_tools.py

```diff
@@ -150,10 +149,15 @@ def test_generate_controller_routes(self):
 
 class RequestLoggingToolTest(ControllerTestCase):
 
-    def __init__(self, *args, **kwargs):
-        cherrypy.tools.request_logging = RequestLoggingTool()
-        cherrypy.config.update({'tools.request_logging.on': True})
-        super(RequestLoggingToolTest, self).__init__(*args, **kwargs)
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        super().enable_request_logging()
```

Comment:
Ok, I see how you managed to work around the issue... But if a third class needs to enable request logging we'd need to invent a third way to call the parent class...

Why not trying (again) with enabling the request logging in the parent `setUpClass` and disabling the warning you got then? Something like:
```suggestion
    @classmethod
    def setUpClass(cls):
        super().setUpClass(request_logging=True)  # pylint: disable=whatever-the-error-was (or even better, if possible, in the parent class)
```

And no need of `tearDownClass` logic in the derived class, as the parent `ControllerTestCase` should automatically disable request logging when it was previously enabled.